runtime/src/accounts_db.rs

```diff
@@ -1014,6 +1015,16 @@ struct CleanKeyTimings {
     dirty_ancient_stores: usize,
 }
 
+/// Fundamental details of an account
+#[derive(Clone, Copy, Debug, Default, Deserialize, PartialEq, Serialize)]
+pub struct AccountDetails {
```

Comment:
This currently excludes account data; I think including account data could make this file quite large, and may or may not necessarily be the most useful.

Given an account that differs, someone should be able to hone in on a TX. My experience working with divergances is that once we have a TX isolated, other clues such as transaction result / lamports / etc typically lead the way to finding the root cause

Reply:
could generate a binary patch from the pre and post account data fields and include that instead?

Reply:
If there was a way to optimally enable the account data, we'd be able to at least turn it on via the ledger-tool in replay?

Reply:
> could generate a binary patch from the pre and post account data fields and include that instead?

At the point we're calling this, the `Bank` is already frozen. So, I'm not sure we could access the pre data in a guaranteed way, maybe peak back at the parent's `Bank` ?

> If there was a way to optimally enable the account data, we'd be able to at least turn it on via the ledger-tool in replay?

Yeah, if we end up not including data by default, I think bubbling up a flag to also include the data would be reasonable